public/controllers/agent/components/register-agent.js

```diff
@@ -221,8 +221,19 @@ export const RegisterAgent = withErrorBoundary(
     async getRemoteInfo() {
       try {
         const result = await WzRequest.apiReq('GET', '/agents/000/config/request/remote', {});
```

Comment:
**nitpick**: If I am not wrong, this request gets the data of the `master` node. Related to this pull request https://github.com/wazuh/wazuh-kibana-app/pull/4737 that adds the selection of the manager to connect, the protocol should use the configurated by the selected manager.

For another hand, what should be the value of the protocol variable for the next situation (taking into account the mentioned pull request):
- manager1: tcp, udp
- manager2: udp

Reply:
Added comment in the linked issue to tread this topic: [https://github.com/wazuh/wazuh-kibana-app/issues/4318#issuecomment-1295303570](https://github.com/wazuh/wazuh-kibana-app/issues/4318#issuecomment-1295303570)